cloud_images/centos7/install_prereqs.sh

```diff
@@ -50,10 +94,10 @@ echo ">>> Removing tty requirement for sudo"
 sed -i'' -E 's/^(Defaults.*requiretty)/#\1/' /etc/sudoers
 
 echo ">>> Install Docker"
-curl -fLsSv --retry 20 -Y 100000 -y 60 -o /tmp/docker-engine-1.13.1-1.el7.centos.x86_64.rpm \
-  https://yum.dockerproject.org/repo/main/centos/7/Packages/docker-engine-1.13.1-1.el7.centos.x86_64.rpm
-curl -fLsSv --retry 20 -Y 100000 -y 60 -o /tmp/docker-engine-selinux-1.13.1-1.el7.centos.noarch.rpm \
-  https://yum.dockerproject.org/repo/main/centos/7/Packages/docker-engine-selinux-1.13.1-1.el7.centos.noarch.rpm
+curl -fLsSv --retry 20 -Y 100000 -y 60 -o /tmp/docker-engine-1.12.6-1.el7.centos.x86_64.rpm \
+  https://yum.dockerproject.org/repo/main/centos/7/Packages/docker-engine-1.12.6-1.el7.centos.x86_64.rpm
```

Comment:
https://docs.mesosphere.com/1.10/installing/oss/custom/system-requirements/install-docker-centos/ states `1.13.1`; why are we reverting?